client-service/src/main/java/com/registry/clientservice/controller/ClientController.java

```diff
@@ -15,17 +17,20 @@
 
     @GetMapping("/{clientNumber}")
     public ClientDTO getClientByClientNumber(@PathVariable("clientNumber") String clientNumber) {
+        log.info("Inside getClientByClientNumber method");
```

Comment:
Adding logs in controller methods is not really a common practice, but I don't see any negatives to it. It would be better to log whatever parameters where passed to the controller method as well.